src/main/java/Deadline.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Deadline Class.
+ * Deadline Class is a subclass of Task Class. It stores the description of this instance
+ * as well as the date and time.
+ *
+ * @author Amos Cheong
+ */
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    /**
+     * Get the description of this Deadline object.
+     * @return String description of this Deadline object.
+     */
```

Comment:
Based on the current coding standard, there should be an empty line between the description and parameter sections.